collect_app/src/main/java/org/odk/collect/android/widgets/utilities/ActivityGeoDataRequester.java

```diff
@@ -6,24 +6,23 @@
 import android.location.Location;
 import android.os.Bundle;
 
-import org.javarosa.core.model.FormIndex;
 import org.javarosa.core.model.QuestionDef;
 import org.javarosa.form.api.FormEntryPrompt;
 import org.odk.collect.android.R;
+import org.odk.collect.android.activities.GeoPointActivity;
+import org.odk.collect.android.activities.GeoPointMapActivity;
 import org.odk.collect.android.activities.GeoPolyActivity;
 import org.odk.collect.android.listeners.PermissionListener;
+import org.odk.collect.android.utilities.ApplicationConstants;
 import org.odk.collect.android.utilities.PermissionUtils;
-import org.odk.collect.android.widgets.interfaces.ActivityGeoDataRequester;
+import org.odk.collect.android.utilities.WidgetAppearanceUtils;
+import org.odk.collect.android.widgets.interfaces.GeoDataRequester;
 
 import java.text.DecimalFormat;
 
 import timber.log.Timber;
 
-import static org.odk.collect.android.utilities.WidgetAppearanceUtils.MAPS;
-import static org.odk.collect.android.utilities.WidgetAppearanceUtils.PLACEMENT_MAP;
-import static org.odk.collect.android.utilities.WidgetAppearanceUtils.hasAppearance;
-
-public class GeoWidgetUtils implements ActivityGeoDataRequester {
```

Comment:
I think the `static` helper methods should stay in the `GeoWidgetUtils` class. There's no reason as far as I can see for them to live in `ActivityGeoDataRequester` and having them there creates a dependency between the widgets and the implementation of `GeoDataRequester` which isn't ideal.